packages/defichain-testing/src/utils.ts

```diff
@@ -0,0 +1,236 @@
+import BigNumber from 'bignumber.js'
+import { RegTestContainer, MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { EllipticPair, Elliptic, Bech32, HRP, WIF } from '@defichain/jellyfish-crypto'
+import { RegTest } from '@defichain/jellyfish-network'
+
+type ContainerType = MasterNodeRegTestContainer | RegTestContainer
+
+/**
+ * send utxos to account
+ *
+ * @param {ContainerType} container
+ * @param {number} _amount
+ * @param {UtxosToAccountOptions} [options]
+ * @param {string} [options.address]
+ * @param {Promise<void>}
+ */
+export async function utxosToAccount (
+  container: ContainerType,
+  _amount?: number,
```

Comment:
We can make this non-optional since it's something user have to be cognizant of when they set up the test criteria.

Reply:
I think we can keep certain things brief/(aka non-optional, implicit) while other explicit. If it helps with reading the testing script setup conditions might want to leave it in.

Reply:
At the same time if you don't see the need for us to set a certain variable like e.g. `mintable: true` (since it's always true for testing, you can keep those implicit.

Reply:
yea noted i check all again